### PR TITLE
feat: 🍰 Mobile Footer Menu To Header Menu

### LINKED ISSUE #4732: 🚀 [Feature] Put Footer Content In The Menu On Mobil Phones
## :rocket: Feature Request

Put footer some content in the menu on mobil phones to have more space on the screen.

- fixes #2292 as well, because it's the same

## Solution

- Put the most footer links into the header menu.
- Leave important legacy links in the footer. (ask @Tirokk and @UlfTramsen for further informations and for talk a bout a space saving solution)

Bildschirmfoto 2021-10-18 um 17 12 01

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pull Request

fist step,  if mobile,  footer menu to heder menu

### Issues
 
- fixes #4732 

### Todo

- [x] if #5527 is merged first then adjust this one

### CODE REVIEW

**Comment by ogerly** [comment]
https://user-images.githubusercontent.com/1324583/196665334-33fff2fd-2c05-41f9-b216-ab84b223fa71.mp4

**Comment by ogerly** [comment]
ok ... i solved it. next to the search input in the header the filter button is only visible on the index. if you move in the network then the filter button disappears ... that's why i didn't see it ...

**Comment by ogerly** [comment]
Bildschirmfoto vom 2022-10-20 09-38-02

File: webapp/components/PageFooter/PageFooter.vue [hunk 1]
```diff
@@ -50,4 +50,9 @@ export default {
   margin-left: 0.2rem;
   margin-right: 0.2rem;
 }
+@media only screen and (max-width: 730px) {
+  .ds-footer {
+    display: none;
+  }
+}
```
Comment (Tirokk) [inline webapp/components/PageFooter/PageFooter.vue:57]:
```suggestion
@media only screen and (max-width: 810px) {
  .ds-footer {
    display: none;
  }
}
```

Then the footer menu doeas not appear at the same time as the footer.

**Review by Tirokk — CHANGES_REQUESTED** [review]
This looks very nice 😍

- There is a width where the footer is still there and the hamurger-menu appears already. I made a suggestion in the code how to avoid this.
- The space between the items of the header menu should be the same as in the very good looking footer menu, see screenshot.
- Can we get the search field etc. already in the second row if the hamburger-menu appears, see screen shot. If possible, that would be perfect. If not we can life with it for now.

Bildschirmfoto 2022-10-21 um 07 18 32

**Review by Tirokk — APPROVED** [review]
Hey @ogerly wonderful !!!

Nice how you refactoed and restructured the menu code 😍

I approve 🚀🚀💫💫

**Comment by Tirokk** [comment]
And allow myself to merge, because then I can go more qucikly ahead …
Hope that's okay.

**Pull request merged**